pkg/graph/schema.resolvers.go

```diff
@@ -1883,6 +1883,60 @@ func (r *mutationResolver) UpdateTRBRequest(ctx context.Context, id uuid.UUID, c
 	return resolvers.TRBRequestUpdate(ctx, id, changes, r.store)
 }
 
+// CreateTRBRequestAttendee is the resolver for the createTRBRequestAttendee field.
+func (r *mutationResolver) CreateTRBRequestAttendee(ctx context.Context, input model.CreateTRBRequestAttendeeInput) (*model.TRBRequestAttendeePayload, error) {
+	attendee := &models.TRBRequestAttendee{
+		TRBRequestID: input.TrbRequestID,
+		EUAUserID:    input.EuaUserID,
+		Component:    input.Component,
+		Role:         input.Role,
+		CreatedBy:    appcontext.Principal(ctx).ID(),
+	}
+	createdAttendee, err := r.store.CreateTRBRequestAttendee(ctx, attendee)
+	if err != nil {
+		return nil, err
+	}
+	return &model.TRBRequestAttendeePayload{
+		TrbRequestAttendee: createdAttendee,
+	}, nil
+}
```

Comment:
With the addition of the main TRB request table (the last PR), we've started chucking resolver logic into the `pkg/graph/resolvers` folder (which is part of `package resolvers`).

This lets us more easily test resolver logic - could you move these implementations into their own files in that package and add tests for them?